spec/triggers/new_event_spec.rb

```diff
@@ -234,6 +254,139 @@
       )
     end
 
+    it 'app_instance.create' do
```

Comment:
Seems like create, update and delete have the same output fields, you can just assign it to a variable let(:app_instance_output_fields) and use it, instead of repeating it 3 times.

Reply:
The spec has been updated using `let(:app_instance_output_fields)`.